File: solutions/1236-n-th-tribonacci-number/solution.py
class Solution(object):
    def tribonacci(self, n):
        """
        :type n: int
        :rtype: int
        """
        a=0
        b=1
        c=1
        f=0;
        if(n==0):
            return 0;
        for i in range(n):
            f=a+b+c;
            a=b;
            b=c;
            c=f;
        return a;
        

File: solutions/1236-n-th-tribonacci-number/solution.cpp
class Solution {
public:
    int tribonacci(int n) {
        long int a=0,b=1,c=1,f;
    if(n==0)
        return 0;
    for(int i=0;i<n;i++)
    {
        f=a+b+c;
        a=b;b=c;c=f;
    }
    return a;
    }
};
